configs/platforms/solaris-11-sparc.rb

```diff
@@ -1,10 +1,3 @@
 platform "solaris-11-sparc" do |plat|
-  plat.servicedir "/lib/svc/manifest"
-  plat.defaultdir "/lib/svc/method"
-  plat.servicetype "smf"
-  plat.cross_compiled true
-  plat.vmpooler_template "solaris-11-x86_64"
-  plat.add_build_repository 'http://solaris-11-reposync.delivery.puppetlabs.net:81', 'puppetlabs.com'
-  plat.install_build_dependencies_with "pkg install ", " || [[ $? -eq 4 ]]"
-  plat.output_dir File.join("solaris", "11", "puppet6")
```

Comment:
We need to ship this packages to specific location so i guess this needs to stay.